Updated styles

i-say-young opened the pull request:
## Overview
Begin adding TRMNL-friendly colors and typography.

i-say-young commented:
(pairing with Ryan)

Rebased `main` into this branch for further development, ran `bin/setup`, then attempted to run server:

```
sayyounghan@Says-MacBook-Air-2 terminus % overmind start --port-step 10 --procfile Procfile.dev --can-die assets,migrate
system | Tmux socket name: overmind-terminus-ulY0T3-Pz6hnKnTj0ZWjO
system | Tmux session ID: terminus
system | Listening at ./.overmind.sock
web    | Started with pid 35752...
assets | Started with pid 35753...
poller | Started with pid 35754...
web    | 
web    | 17:36:15 [rerun] Terminus launched
web    | 17:36:15 [rerun] Rerun (35757) running Terminus (35779)
poller | /var/folders/4b/yc_dydws78gcnlgl1f3s6w3m0000gn/T/overmind-terminus-ulY0T3-Pz6hnKnTj0ZWjO/poller: line 4: bin/screen_poller: No such file or directory
poller | Exited with code 127
web    | Interrupting...
assets | Interrupting...
assets | /Users/sayyounghan/.rbenv/versions/3.4.2/lib/ruby/3.4.0/bundled_gems.rb:82:in 'Kernel.require': Interrupt
assets | 	from /Users/sayyounghan/.rbenv/versions/3.4.2/lib/ruby/3.4.0/bundled_gems.rb:82:in 'block (2 levels) in Kernel#replace_require'
... 
...
web    | 
web    | 17:36:15 [rerun] Terminus stopping
assets | Exited with code 0
web    | Exited with code 0
```

bkuhlmann commented:
Oh, nice! A few notes to help you along:

- The setup script is designed to *not* overwrite your existing setup (in case you customize it). So you'd need to delete your existing `Procfile.dev` and run setup again to pick up the latest changes.
- You might want to break your commits into smaller, [atomic](https://alchemists.io/articles/git_commit_anatomy), commits. This'll make rebasing easier but also give you space to make individual fixups for a cleaner Git commit history.
- The build is failing because you don't have a proper Git author name. This is super easy to fix as you only need to edit your [Git Configuration](https://alchemists.io/articles/git_configuration). Like this:

``` 
[user]
  email = "<add your email address here>
  name = Seyoung Kulp
```

Let me know if that helps and if I can help further!

i-say-young commented:
I fixed my git config name and began adding a title + description to commits, but seeing errors for previous commits. Is the only option to `git revert` and then force push? How can we ignore CI failures for previous commits, since latest is correct?

```
2c05b341a1577fa9a65102bf9f15ca912a9ca39c (Seyoung Kulp, 12 minutes ago): Updated dashboard ui
  Commit Body Presence Warning. Use minimum of 1 line (non-empty).

e3f329db0a80a71f8f793d77c223a7d26d139d5e (i-say-young, 31 hours ago): Updated styles
  Commit Author Capitalization Error. Capitalize each part of name: "i-say-young".
  Commit Author Name Error. Author name must consist of 2 parts (minimum).
  Commit Body Presence Warning. Use minimum of 1 line (non-empty).
  ```

bkuhlmann commented:
Nice work. So the issue with the first commit is that it's still using your old author name even though you updated your Git configuration (which is great). This is simple to fix, though. All you need to do is rebase and update your author name for all commits to be the same. The quickest way to do this is:

``` bash
GIT_EDITOR=true git rebase --interactive main --exec 'git commit --amend --no-edit --author "Seyoung Kulp <hana130sy@gmail.com>"'
```

What the above does is:

1. Initiate an interactive rebase except disabled the interactive part via `GIT_EDITOR=true`. A little counterintuitive but powerful.
2. Ensures the rebase only happens with commits that are in your feature branch (i.e. what's new from the `main` branch).
3. Run exec (i.e. `--exec`) which allows Git to run any code during each commit touched during the rebase process. In this case we are simply updating each of your commits to use your new author name and email so they are consistent.

Once you've done this, you'll need to push to your remote branch using `git push --force-with-lease` (i.e. safe push, not destructive push).

That's it! Let me know if that helps.

bkuhlmann commented:
:bulb: If you run the following, this'll reduce the massive amount of confusing commits on this feature branch:

``` bash
git switch main
git pull --rebase
git switch styling
git pull --rebase origin main
git push --force-with-lease
```

bkuhlmann commented:
:sparkle: The new design changes look great and thanks for all the work on this.

Are you ready to fold these changes in? If so, I think there are only a couple tweaks needed (:

- This commit `Updated default device image file type` looks like a fixup to `Added terminal defaut device setup screen`.
- I left a few commits on separate commits.

Would you like me to do this or would you prefer to do this?

bkuhlmann commented on config/db/structure.sql:
```diff
@@ -2,8 +2,8 @@
 
--- Dumped from database version 17.0
--- Dumped by pg_dump version 17.0
+-- Dumped from database version 17.0 (Postgres.app)
```
:axe: These changes are safe to ignore and can be removed since we don't need to capture that the Postgres.app touched this file.

i-say-young replied:
I think I pull the branch and started designing so probably I kept the change. 
I just finished device view page and I think we're ready to fold in. 

Still having issue with text editor that I don't think I can fix up the commits. Just switched from Atom to Sublime Text but because of my computer set up it's having some issues. 
If you can do this one that would be great. For the future commit, I'll figure out with my computer.

bkuhlmann commented:
ℹ️  OK, I'm going to fold in these changes by rebasing and taking out the noisy commits so the history remains clean and easy to read. Once done, I'll fold these changes onto `main`.

bkuhlmann commented:
:information_source: Rebase complete. Checking build and remaining commits to make sure I didn't miss anything.
